src/neo/SmartContract/Native/Tokens/Nep17Token.cs

```diff
@@ -67,10 +67,10 @@ internal protected virtual void Mint(ApplicationEngine engine, UInt160 account,
             state.Balance += amount;
             storage = engine.Snapshot.Storages.GetAndChange(CreateStorageKey(Prefix_TotalSupply), () => new StorageItem(BigInteger.Zero));
             storage.Add(amount);
-            PostTransfer(engine, null, account, amount);
+            PostTransfer(engine, null, account, amount, callOnPayment);
         }
 
-        internal protected virtual void Burn(ApplicationEngine engine, UInt160 account, BigInteger amount)
+        internal protected virtual void Burn(ApplicationEngine engine, UInt160 account, BigInteger amount, bool callOnPayment)
```

Comment:
It is impossible to call `callOnPayment` in `Burn` because `to` is always `null`.

Reply:
Done